### PR TITLE
feat: set the preferTranscoding param correctly which is subtask of SPARK-379327

### PR DESCRIPTION
**Pull request by x-epoch**
# COMPLETES # https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-379327

## This pull request addresses

the code in sendRoapTurnDiscoveryRequest() in turnDiscovery.ts in JS-SDK needs to be updated to set the preferTranscoding param correctly when calling sendRoap()

## by making the following changes

< DESCRIBE YOUR CHANGES >

### Change Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios where tested

Unit test in local, by running the yarn test:unit,  all `doTurnDiscovery ` related cases passed

### I certified that

- [ ] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [ ] I discussed changes with code owners prior to submitting this pull request

- [ ] I have not skipped any automated checks
- [ ] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/plugin-meetings/src/meeting/index.ts [hunk 1]
```diff
@@ -469,7 +469,9 @@ export default class Meeting extends StatelessWebexPlugin {
   resourceId: any;
   resourceUrl: string;
   selfId: string;
-  state: any;
+  state: any;  
+  isMultistream: boolean;
```
Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:473]: I think you meant to do this change on a branch from "beta" branch and merge this PR into "beta" branch, not master. We don't have the isMultistream parameter on SDK master

Comment (marcin-bazyl) [inline packages/@webex/plugin-meetings/src/meeting/index.ts:473]: I think you meant to do this change on a branch from "beta" branch and merge this PR into "beta" branch, not master. We don't have the isMultistream parameter on SDK master

**Review by marcin-bazyl — CHANGES_REQUESTED** [review]

File: packages/@webex/plugin-meetings/src/meeting/index.ts (hunk 1 above)
Comment (x-epoch) [reply packages/@webex/plugin-meetings/src/meeting/index.ts:473]: do you mean we have 2 branch, one named `master` for transcoding meeting and other other named `beta` for multiple stream meeting ? Further more, I should create a new pr to merge to code changes to `beta` but not `master`, correct ?

**Comment by x-epoch** [comment]
replaced by https://github.com/webex/webex-js-sdk/pull/2558

**Pull request closed without merging**